standup/auth0/pipeline.py

```diff
@@ -33,13 +36,14 @@ def run_pipeline(pipeline, **kwargs):
     return kwargs
 
 
-def log_user_info(user_info, **kwargs):
-    """Logs user_info
+def log_info(user_info, token_info, **kwargs):
```

Comment:
I should generalize this to just print everything in `kwargs`. Then you can use it to debug the pipeline.